.github/workflows/functionality.yml

```diff
@@ -72,41 +69,6 @@ jobs:
             bash -x "$runscript";
           done
 
-  sda-pipeline:
```

Comment:
I don't see where we do functionality/integration testing of sda (formerly sda-pipeline) now?

Reply:
They are triggerede here: https://github.com/neicnordic/sensitive-data-archive/blob/80f8fbf2326fc860fd2c1620afb7c32f4959834e/.github/workflows/build_pr_container.yaml#L226

Reply:
Thanks.